adal-analytics/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,7 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
```

Comment:
Because when you call the FirebaseAnalytics.getInstance() the IDE shows a warning with: "Missing permissions required by FirebaseAnalytics.getInstance: android.permission.WAKE_LOCK"